apps/prairielearn/elements/pl-drawing/elements.py

```diff
@@ -1727,6 +1727,51 @@ def get_attributes():
         ]
 
 
+class Coil(BaseElement):
+    def generate(el, data):
+        stroke_color = pl.get_color_attrib(el, "stroke-color", "black")
+        x1 = pl.get_float_attrib(el, "x1", drawing_defaults["x1"])
+        y1 = pl.get_float_attrib(el, "y1", drawing_defaults["y1"])
+        if "x2" in el.attrib and "y2" in el.attrib:
```

Comment:
Is this the same style used in other places in this file? Not sure if `pl.has_attrib` is more appropriate.

Reply:
It is the same style. I just copied the code from other similar elements (for example Spring, Line, etc)